src/Path.hs

```diff
@@ -279,18 +287,18 @@ isProperPrefixOf p l = isJust (stripProperPrefix p l)
 --
 -- The following properties hold:
 --
--- @
--- parent (x \<\/> y) == x
--- parent \"\/x\" == \"\/\"
--- parent \"x\" == \".\"
--- @
+-- @parent (x \<\/> y) == x@
 --
--- On the root (absolute or relative), getting the parent is idempotent:
```

Comment:
Two reasons:

1) To keep that comment I have to group the root related properties together, instead I wanted to group relative and absolute properties separately. It also makes the `parent` and `dirname` properties format the same.

2) It is obvious from the rules.